ROADMAP.md

```diff
@@ -0,0 +1,54 @@
+- 📪 Implement archiving of old builds: the yaml spec and the lockfile are kept, but the environments on disk are deleted.
+- 📪 Implement bulk archiving of old builds. --all-but-last, --older-than=<date>.
+- 📪 Implement recreation of old builds. Combined with planned performance improvements, users can expect to wait only as long as it takes to download and reinstall the packages for the recreated environment.
+- 📪 Transferring or copying a full conda-store deployment is currently not possible. Implement full backup and restore, including creating the active environments on disk.
+- 📪 Currently, conda environments created by conda-store do not share packages. In deployments containing many builds, this can lead to many duplicate packages, which uses much more storage than necessary. Refactor the logic around creating new environments so that packages are shared across environments.
+- 📪 Design a serialization format for conda-store environments that captures both the history of specfiles and lockfiles to allow transferring and copying of individual environments.
+
+## User experience improvements
+
+- 🏃 Currently there are three ways to interact with the conda-store-server: the CLI, the REST API, and the Python API. However, these interfaces do not provide the same functionality, leading to maintenance issues. Addressing this will require refactoring the existing CLI.
+- ✅ Determine user workflows to scope CLI functionality
+- 📪 Design the new CLI
+- 📪 Implement the new CLI
+- 📪 The current conda-store-ui has proven difficult to extend, improve, and maintain. A refactor to simplify and standardize its codebase, while following industry best-practices is currently being proposed which includes updates to the UI to match the new workflows mentioned above.
+- 📪 Ensure parity between the CLI, REST API, and the Python API
+- 🏃 Additional documentation for developers is critical for helping potential contributors, but our current developer documentation does not cover all parts of the code.
+- 🏃 Existing architectural diagrams give an overview of what microservices comprise a complete conda-store deployment. Additional documentation would be useful, particularly to help newcomers understand what conda-store is doing when the user takes various actions.
+- 📪 Additional documentation about authorization and authentication
+- 📪 Refactor jupyterlab-conda-store to make it a simple iframe of the existing UI, with cross-iframe messages + URL query parameters for communication and customization.
+- 📪 Instead of requiring the user to set an environment variable to build GPU-enabled environments, explicitly add this option to conda-store-ui to make this process smoother.
```

Comment:
```suggestion
- 📪 Instead of requiring the user to set an environment variable to build GPU-enabled environments, explicitly add this option to conda-store to make this process smoother.
```
this has to be done both on conda-store and conda-store-ui (the representation)